tool-populate/api/helpers.js

```diff
@@ -0,0 +1,21 @@
+var Parse = require('parse/node')
+/*
+  Accepts Parse object and sets ACL giving users in Admin Role read/write access
+  @param [Object] obj, Parse object on which to modify ACL
+ */
+function setAdminACL(obj) {
+  var query = new Parse.Query(Parse.Role)
+  query.equalTo('name', 'admin')
+  return query.first({useMasterKey: true})
+    .then(function(adminRole) {
+      var acl = new Parse.ACL()
+      acl.setPublicReadAccess(false)
+      acl.setPublicWriteAccess(false)
+      acl.setReadAccess(adminRole, true)
+      acl.setWriteAccess(adminRole, true)
+      obj.setACL(acl)
```

Comment:
@jgoley I still think you need to return the obj after the acl has been set, otherwise its a no-op.  All the questions are still Public Read + Write.

![image](https://cloud.githubusercontent.com/assets/12954045/15944195/6d92329e-2e5a-11e6-81de-6f2d2cb1f788.png)